src/game.js

```diff
@@ -1151,19 +1165,21 @@ export default class Game {
 		return damage;
 	}
 
-	onDamage(creature, damage) {
+	// creature, damage -- removed because used arguments.
```

Comment:
Small insignificant comment inconsistency: some have ending dots, some don't. Maybe no dots for all of them...